tests/sparseml/pytorch/sparsification/pruning/test_mfac.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import torch
+
+from sparseml.pytorch.sparsification.pruning import (
+    FisherInverseFast,
+    FisherInverseFastBlock,
+    FisherInverseFastSmallBlocks,
+)
+
```

Comment:
can we use `pytest.mark.parameterize` to reuse test code here? if for some reason we can't, let's move some of the shared code to a helper function